pkg/gator/oci.go

```diff
@@ -0,0 +1,90 @@
+package gator
```

Comment:
let's move this to its own package... `gator` is getting a little "kitchen-sinky"